src/Raven.Client/Documents/Operations/AI/AiUsage.cs

```diff
@@ -25,6 +25,14 @@ internal void UpdateFrom(BlittableJsonReaderObject json)
             if (promptDetails.TryGet("cached_tokens", out int cachedTokens))
                 CachedTokens += cachedTokens;
         }
+
+        if ((json.TryGet("completion_tokens_details", out BlittableJsonReaderObject completionTokensDetails) || completionTokensDetails == null) == false)
+            return;
+        if ((completionTokensDetails != null && completionTokensDetails.TryGet("reasoning_tokens", out int reasoningTokens)) == false)
```

Comment:
I'm _pretty_ sure that this is still something that as a user, you pay for, no?
That is charged the same as output tokens, and as such, something that should be accounted.